cfgov/unprocessed/apps/regulations3k/css/main.less

```diff
@@ -5,6 +5,7 @@
 
 
 @import (less) '../../../css/main.less';
+@import (less) './notification.less';
```

Comment:
This is very minor, but should we call this `reg-notification.less` to stay consistent with the other regulations-specific Less we've created?

Reply:
I didn't because... inherently there's nothing specific to regulations3k about this. It could easily be moved into v1 and used on other page types as well. So, I figured I'd make it as generic as possible so that that option remains.

Reply:
Gotcha, that makes sense to me.